rules/integrations/aws/exfiltration_ec2_snapshot_change_activity.toml

```
@@ -23,7 +23,63 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "AWS EC2 Snapshot Activity"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating AWS EC2 Snapshot Activity
+
+Amazon EC2 snapshots are a mechanism to create point-in-time references to data that resides in storage volume. System
```

Fixing subject-verb agreement, and also "storage volume" prob either needs to be plural or add an article ("reside in _a_ storage volume"):
```suggestion
Amazon EC2 snapshots are a mechanism to create point-in-time references to data that reside in storage volumes. System
```